src/app/Main/Body/InfoPage/InfoPage-Container.component.tsx

```diff
@@ -0,0 +1,333 @@
+        t("Methods.Amrs.TableHeaderAmrSubstance"),
+        t("Methods.Amrs.TableHeaderCuttOffFaecalis"),
+        t("Methods.Amrs.TableHeaderCuttOffFaecium"),
+        t("Methods.Amrs.TableHeaderMin"),
+        t("Methods.Amrs.TableHeaderMax"),
+    ];
+
+    const amrKeys: AmrKeyType[] = [
+        "coliSalm",
+        "coliSalmTwo",
+        "campy",
+        "mrsa",
+        "ef",
+    ];
+
+    const tableDataAmrColiSalm: TableData = {
+        title: t("Methods.Amrs.coliSalm.TableTitle"),
+        description: t("Methods.Amrs.coliSalm.TableDescription"),
+        tableHeader: coliSalmTableHeader,
+        tableRows: tableAmrEcoliSalmRows,
+        commentText: t("Methods.Amrs.coliSalm.TableComment"),
+    };
+    const tableDataAmrColiSalmTwo: TableData = {
+        title: t("Methods.Amrs.coliSalmTwo.TableTitle"),
+        description: t("Methods.Amrs.coliSalmTwo.TableDescription"),
+        tableHeader: coliSalmTableHeader,
+        tableRows: tableAmrEcoliSalmTwoRows,
+        commentText: t("Methods.Amrs.coliSalmTwo.TableComment"),
+    };
+    const tableDataAmrCampy: TableData = {
+        title: t("Methods.Amrs.campy.TableTitle"),
+        description: t("Methods.Amrs.campy.TableDescription"),
+        tableHeader: oneCutOffTableHeader,
+        tableRows: tableAmrCampyRows,
+        commentText: t("Methods.Amrs.campy.TableComment"),
+    };
+    const tableDataAmrMrsa: TableData = {
+        title: t("Methods.Amrs.mrsa.TableTitle"),
+        description: t("Methods.Amrs.mrsa.TableDescription"),
+        tableHeader: oneCutOffTableHeader,
+        tableRows: tableAmrMrsaRows,
+        commentText: t("Methods.Amrs.mrsa.TableComment"),
+    };
+    const tableDataAmrEf: TableData = {
+        title: t("Methods.Amrs.ef.TableTitle"),
+        description: t("Methods.Amrs.ef.TableDescription"),
+        tableHeader: efTableHeader,
+        tableRows: tableAmrEfRows,
+        commentText: t("Methods.Amrs.ef.TableComment"),
+    };
+
+
+    const amrTableData: Record<AmrKeyType, TableData> = {
+        coliSalm: tableDataAmrColiSalm,
+        coliSalmTwo: tableDataAmrColiSalmTwo,
+        campy: tableDataAmrCampy,
+        mrsa: tableDataAmrMrsa,
+        ef: tableDataAmrEf
+    };
+
+
+
+    const handleExportAmrData = (amrKey: AmrKeyType): void => {
+        const csvHeader = amrTableData[amrKey].tableHeader.join(",");
+
+        let csvContent = "";
+        amrTableData[amrKey].tableRows.forEach((tableRow) => {
+            const rowValues = Object.values(tableRow);
+            const modifiedRowValues = rowValues.map((rowValue) =>
+                modifyTableDataStringService(rowValue)
+            );
+            const modifiedRowValuesString = modifiedRowValues.join(",");
+            csvContent += modifiedRowValuesString;
+            csvContent += "\n";
+        });
+        const csvTable = `${csvHeader}\n${csvContent}`;
+        const amrTableExportElement = document.createElement("a");

```

Comment:
Another possibility would be the CSVLink from react-csv, which I used in the header-export. But you didn't seem to be happy with that one either?

Reply:
I'm unhappy with both solutions :) But this might be the better one. I really don't know if there is a good solution, i don't implemented an export file functionality yet in javascript. But you can ask e.g. Marco about their export button in FCL or just leave it as it is.